app/models/manageiq/providers/openshift/container_manager_mixin.rb

```diff
@@ -32,10 +32,10 @@ def raw_connect(hostname, port, options)
     end
```

Comment:
Not for this PR but:

It's weird to me that we both provided a new `openshift_connect` method AND overrode the *default* behaviour of `raw_connect` (the 3 lines above, super [here][1]) to call `openshift_connect`.
The openshift `/oapi` client is not a drop-in replacement for a kubernetes client, right?  Then modifying the default might break superclass code, and it'd be better to keep default to kubernetes and have openshift code always explicitly request openshift client.  (Or even have all code explicitly specify :service)

https://github.com/ManageIQ/manageiq/blob/3d42ae360d38f048dbc2e6e79e67cb9d01666ab7/app/models/manageiq/providers/kubernetes/container_manager.rb#L44-L46